inst/extdata/gcam-china/CCS_provincial_data_add.csv

```diff
@@ -0,0 +1,19 @@
+# File: CCS_provincial_data_add.csv
+# Title: CO2 and Cost(1990USD) by province
+# Units: CO2-Mt,Cost-1990USD per t CO2
```

Comment:
Units here don't match the ones in variable names. Double check and make sure you use correct units.

Reply:
get